crowd-forecast/app.R

```diff
@@ -31,6 +31,7 @@ setorder(observations, target_type, target_end_date)
 run_app(data = observations, 
         first_forecast_date = as.character(first_forecast_date),
         selection_vars = c("location_name", "target_type"),
+        path_service_account_json = ".secrets/crowd-forecast-app-c98ca2164f6c-service-account-token.json",
```

Comment:
have you put this in the `.secrets` folder for each app (i.e in their own folders)? From how I think this is working you need to?

Reply:
yep so currently the secrets are copied three times: once for the overall repo and once for every app

Reply:
cool that is what I thought.